en/docs/testing/test-rest-endpoints-via-the-openapi-console.md

```diff
@@ -0,0 +1,32 @@
+# Test REST Endpoints via the OpenAPI Console
+
+Devant offers an integrated OpenAPI Console to test REST endpoints for the Service components you create and deploy. Since Devant secures REST APIs with OAuth 2.0 authentication, the OpenAPI Console generates test keys to help you test your APIs.
+
+Follow these steps to test a REST endpoint using the OpenAPI Console:
+
+1. Go to the [Devant Console](https://console.devant.dev/) and log in.
+2. Select the project.
+3. In the **Integrations** pane, click on the component you want to test.
+4. Click **Test** in the left navigation menu, then select **Console**. This will open the **OpenAPI Console** pane.
+5. In the **OpenAPI Console** pane, select the desired environment from the drop-down menu.
+6. Choose the endpoint you want to test from the **Endpoint** list.
+
+    !!! note
+        - The **Network Visibility** of an endpoint is configured in the **component.yaml** file. You can set it to **Public**, **Organization**, or **Project**.
```

Comment:
```suggestion
        - The **Network Visibility** of an endpoint is configured in the **component.yaml** file. You can set it to `Public`, `Organization`, or `Project`.
```
Values should be as code. Update the other places as well

Reply:
Fixed via [4dcea89](https://github.com/wso2/docs-devant/pull/59/commits/4dcea89c85fbc5f21aa7769a876bdc0bb3e7ab60)